tracpro/templates/baseline/baselineterm_chart.html

```diff
@@ -0,0 +1,47 @@
+<script src="http://code.highcharts.com/modules/exporting.js"></script>
+<script>
+$(function () {
+    $('#container').highcharts({
+        title: {
+            text: '{{ baselineterm.name|escapejs }}'
+        },
+        subtitle: {
+            text: 'Baseline: {{ baselineterm.baseline_question|escapejs }}, Follow Up: {{ baselineterm.follow_up_question|escapejs }}'
+        },
+        xAxis: {
+            categories: [
+                        {% for date in date_list %}
+                            '{{ date }}',
```

Comment:
I love trailing commas when splitting things over multiple lines in Python. But Javascript doesn't always like them. Maybe the comma can be surrounded by `{% if not forloop.last %},{% endif %}`?